api/python/quilt3/main.py

```diff
@@ -46,6 +46,40 @@ def create_parser():
     )
     config_p.set_defaults(func=cmd_config)
 
+    shorthelp = "Install a package"
+    install_p = subparsers.add_parser("install", description=shorthelp, help=shorthelp)
+    install_p.add_argument(
+        "name",
+        help="Name of package, in the user/pkg format",
```

Comment:
In docs we usually use all caps for template strings.
```suggestion
        help="Name of package, in the USER/PKG format",
```